### DISCUSSION TITLE
Environment size

### DISCUSSION THREAD

**Discussion by j2969719** [dosemu2/dosemu2#2184] [Q&A]
A bit dumb question, but is it possible for a user to increase the environment size when using fdpp? For older versions of dosemu2 I simply used the shotgun approach and patched shellhigh in src/bindist/fdppconf.sys but this won't work anymore.

**Comment by stsp** [comment]
Why do you need this?
Are you using some other
command.com than comcom32/64?
Because with these command.com's
the real-mode env block is mostly
unneeded.

**Comment by j2969719** [comment]
Yes, in one old drive_c folder I have freecom in the root and too many variables in userhook.bat.
